reftests/src/config/ledger.json

```diff
@@ -0,0 +1,4 @@
+{
+    "faucet": "maffbahksdwaqeenayy2gxke32hgb7aq4ao4wt745lsfs6wijp",
+    "symbol": "mqbfbahksdwaqeenayy2gxke32hgb7aq4ao4wt745lsfs6wiaaaaqnz"
```

Comment:
We should probably put the key here. We should not have to modify anything to run those tests.